wizeline/project-lab
# Feature/unified-create-form 68

## Feat - 68 - Simplify Creation Flow

#### What does this PR do?
- Merge both type of proposal creation on same page, toggling between form types.
- Adjust form elements styles for alignment and width.
- Redirect create button route to form create.
- Show form switch button only on create page, and show full form on edit page


#### Where should the reviewer start?
1. app/pages/projects/full.tsx  // This is the creation page for both types (quick / detail)
2. app/projects/components/ProjectForm.tsx // The form component with the switch feature, it's text and select elements have added attributes in order to style them to full width 

#### How should this be manually tested?

1. Go to http://localhost:3000/projects/ and create a new project.
2. The page will take you directly to the creation page, with a switch button to show/hide fields.
3. When editing a proposal the switch button will not appear, but the form will be a full form

#### Any background context you want to provide?
This change allows to create and edit proposals with the same component, adding props for the specific use

#### What are the relevant tickets?
This closes ticket #68 

#### Screenshots



#### Questions
I'll appreciate any comments about clean-up and format, or any other question. Also if it's ok to remove the comments of changes made.

#### Checklist
